cli/cmd/suppressions_gcp.go

```diff
@@ -58,3 +156,213 @@ func suppressionsGcpList(_ *cobra.Command, _ []string) error {
 	}
 	return cli.OutputJSON(suppressions)
 }
+
+func suppressionsGcpMigrate(_ *cobra.Command, _ []string) error {
+	var (
+		suppressionsMap map[string]api.SuppressionV2
+		err             error
+
+		convertedPolicyExceptions []map[string]api.PolicyException
+		payloadsText              []string
+		discardedSuppressions     []map[string]api.SuppressionV2
+	)
+	suppressionsMap, err = cli.LwApi.V2.Suppressions.Gcp.List()
+	if err != nil {
+		if strings.Contains(err.Error(), "No active GCP accounts") {
+			cli.OutputHuman("No active GCP accounts found. " +
+				"Unable to get legacy gcp suppressions")
+			return nil
+		}
+		return errors.Wrap(err, "Unable to get legacy gcp suppressions")
+	}
+
+	if len(suppressionsMap) == 0 {
+		cli.OutputHuman("No legacy GCP suppressions found.\n")
+		return nil
+	}
+
+	answer := ""
+	manualMigration := "Output translated legacy suppressions as policy exception commands to be" +
+		" run manually (Recommended)"
+	autoMigration := "Auto migrate legacy suppressions.\nDISCLAIMER: " +
+		"By selecting this option, you accept liability for the migration and " +
+		"any compliance violations missed as a result of the added exceptions"
+	if err := SurveyQuestionInteractiveOnly(SurveyQuestionWithValidationArgs{
+		Prompt: &survey.Select{
+			Message: "Select your legacy suppression migration approach?",
+			Options: []string{
+				manualMigration,
+				autoMigration,
+			},
+		},
+		Response: &answer,
+	}); err != nil {
+		return err
+	}
+
+	// get a list of all policies and parse the valid exception constraints and create a map of
+	// {"<policyId>": [<validPolicyConstraints>]}
+	policyExceptionsConstraintsMap := getPoliciesExceptionConstraintsMap()
+
+	switch answer {
+	case manualMigration:
+		_, payloadsText, discardedSuppressions = convertGcpSuppressions(
+			suppressionsMap,
+			policyExceptionsConstraintsMap,
+		)
+		printPayloadsText(payloadsText)
+		printDiscardedSuppressions(discardedSuppressions)
+	case autoMigration:
+		convertedPolicyExceptions, _, discardedSuppressions = convertGcpSuppressions(
+			suppressionsMap,
+			policyExceptionsConstraintsMap,
+		)
+		printConvertedSuppressions(convertedPolicyExceptions)
+		confirm := false
+		err := survey.AskOne(&survey.Confirm{
+			Message: "Confirm the above exceptions have been reviewed and you wish to continue" +
+				" with the auto migration.",
+		}, &confirm)
+		if err != nil {
+			return err
+		}
+		if confirm {
+			autoConvertSuppressions(convertedPolicyExceptions)
+			printDiscardedSuppressions(discardedSuppressions)
+			cli.OutputHuman(color.GreenString("To view the newly created Exceptions, " +
+				"try running `lacework policy-exceptions list <policyId>"))
+		} else {
+			cli.OutputHuman("Cancelled Legacy Suppression to Exception migration!")
```

Comment:
nit, need a new line here to stop % being output
```suggestion
			cli.OutputHuman("Cancelled Legacy Suppression to Exception migration!\n")
```